src/iexamine.cpp

```diff
@@ -2233,7 +2233,7 @@ void iexamine::keg(player &p, const tripoint &examp)
             add_msg(m_info, _("You don't have any drinks to fill the %s with."), g->m.name(examp).c_str());
             return;
         }
-        // Make lists of unique drinks... about third time we do this, maybe we oughta make a function next time
```

Comment:
`wanna` is ok, but not `oughta` ;)

Reply:
I don't wanna "oughta", but I ought to replace "want to". ;)